report/src/main/webapp/app/themes/winterEdition.directive.js

```diff
@@ -43,8 +43,10 @@ define(['angularAMD'], function (angularAMD) {
       var isChristmas = isDateWithinMargin(TODAY, CHRISTMAS_DATE, DAY_MARGIN);
 
       if (isChristmas) {
```

Comment:
Could we please add additional flag, e.g. `winterEditionEnabled` and make the condition:
```
if (winterEditionEnabled && isChristmas) {
```
?
It should be set to `false` by default.
If admin of an AET instance decides to enable it, it should be manually set to `true` on the server instance.

Reply:
Done.